main.css

```diff
@@ -108,3 +121,2555 @@ body {
       font-size: 0.875rem; /* 14px */
       font-weight: 600;
     }
+
+.prog-bar{}
+  .prog-bar__button{
+    margin: 0.75rem 1.5rem;
+  }
+  .prog-bar__button-dropdown{
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    padding: 0.5rem 2.5rem;
+    margin: 0.75rem 1.5rem;
+    font-size: 0.8125rem;
+    background: none;
+    border: none;
+    background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACYAAAAgCAYAAAB+ZAqzAAAAAXNSR0IArs4c6QAAAVhJREFUWAnNkK8vRWEch49hNpsgCIoiSZIm+hNEURQEURRFURTF20RNkiRFUWyCzYYZnod7bcxx7jnvr/Nsn3vfe877/Xyf3aqqqgF571l0qpbJU4/EdNHpkwM++/Kv6fLNLKcbUlpOB11+sMmv0mI6/MkZT0vJubuWFd68kNxy7nT3vxzyNreYOxuZ48YtySXnLneOxRa3com5qxXn3E4t547WrDLxSlLJ2e2OThwxlUrM7s7MM3lHYsvZaXcQ20zHFrMzmAkaLkgsObvsjMIaLW8kVM4Ou6JyTFuomB3RWaDxnnSVc9aOJOzQ2lXM2WRM0nxJ2so542xS1mlvK+ZMFk7YMq6cd7OxyKYH0iTnHe9mZY9tTWLeyc40G69InZzvvFOEDbbWifmuKKds/y3ns+IsYfBIRnKefdYL9rEYiXnuDTOYXA/jOZip4Iavgme+doddnoP5AEOdX53zUZ1XAAAAAElFTkSuQmCC);
+    background-size: 0.625rem 0.5rem; /* 10px 8px */
+    background-repeat: no-repeat;
+    background-position: 95%;
+    cursor: pointer;
+  }
+      .prog-bar__button-dropdown--active{
+        border-bottom: solid 2px #003066;
```

Comment:
Let's change this to:
```css
-webkit-box-shadow: 0px 2px 0px 0px #003066;
-moz-box-shadow: 0px 2px 0px 0px #003066;
box-shadow: 0px 2px 0px 0px #003066;
```
This will prevent moving all of the other content further down the page by 2px and **only** highlight the dropdown button